src/Stripe.net/Services/Account/StripeAccountCreateOptions.cs

```diff
@@ -7,7 +7,7 @@ public class StripeAccountCreateOptions : StripeAccountSharedOptions
         [JsonProperty("country")]
         public string Country { get; set; }
 
-        [JsonProperty("managed")]
-        public bool? Managed { get; set; }
+        [JsonProperty("type")]
+        public string Type { get; set; }
```

Comment:
No worries with conflicts here (.net uses `GetType` and `typeof` as well) . You'll see various places i've had the same reservations for properties like this, Id, and Name. Since this is in the context of an AccountCreateOptions, I think it's intuitive that the type is for Account. 👍